README.md

```diff
@@ -188,7 +189,25 @@ storyblok delete-components <SOURCE> --space <SPACE_ID>
 ```
 
 #### Parameters
-* `source`: can be a URL or path to JSON file.
+* `source`: can be a URL or path to JSON file, the path to a json file could be to a single or multiple files separated by comma, like `./pages-1234.json,../User/components/grid-1234.json`
+
+Using an **URL**
+
+```sh
+$ storyblok push-components https://raw.githubusercontent.com/storyblok/nuxtdoc/master/seed.components.json --space 67819
+```
+
+Using a **path** to a single file
+
+```sh
+$ storyblok push-components ./components.json --space 67819
+```
+
+Using a **path** to a multiple files
+
+```sh
+$ storyblok push-components ./page.json,../grid.json,./feature.json --space 67819
+```
```

Comment:
The PR is from delete components and this section is from push-components, it's incorrect, right ?